contracts/cw1-subkeys/helpers.ts

```diff
@@ -135,9 +135,17 @@ const useOptions = (options: Options): Network => {
 
 type Expiration = { at_height: { height: number } } | { at_time: { time: number } } | { never: {}}
 
+interface Permissions {
```

Comment:
Question:

If we add permissions for eg. governance votes, or calling wasm contracts, would you keep them here? Or use a separate type? 

To see if we want one large "Permissions" or "StakingPermissions", "VotingPermissions", etc.

Reply:
Separate permissions design brings a lot of extra code(bucket) to both smart contract and helpers.ts. This struct will not contain hundreds of booleans(max 15-20?). I guess using single permissions struct is the optimal way

Reply:
If rust and buckets provides a way to define permissions generically, then multiple permissions that extend `Permissions` trait is better.

Reply:
Fair enough, we keep one Permissions and extend it for other packages.
Please add a comment on the type explaining that choice (actually only in helpers.rs, not here)

Reply:
I assume you mean state.rs by helpers.rs? or helpers.ts? I am confused 😕

Reply:
I meant `state.rs`
I should re-review again